src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -2865,11 +2969,25 @@ void Goo()
     {
         var t1 = new N.NewStruct(a: 1, b: 2);
     }
-}
-        </Document>
-    </Project>
-</Workspace>";
-            await TestInRegularAndScriptAsync(text, expected, index: 2, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+}";
+
+            var test = new VerifyCS.Test
+            {
+                CodeActionIndex = 2,
+                CodeActionEquivalenceKey = null,
```

Comment:
📝 All tests that set `CodeActionIndex` need to also set a non-null `CodeActionEquivalenceKey`. You may need to fix the refactoring implementation to set this.

Reply:
have done this now.